dask/array/utils.py

```diff
@@ -121,6 +125,21 @@ def assert_eq(a, b, check_shape=True, check_graph=True, **kwargs):
 
     try:
         assert a.shape == b.shape
+        if check_meta:
+            if hasattr(a, '_meta') and hasattr(b, '_meta'):
+                assert a._meta.ndim == b._meta.ndim
```

Comment:
I think that you should be able to do `assert_eq(a._meta, b._meta)` to check a variety of things

Reply:
Indeed, done in https://github.com/dask/dask/pull/4543/commits/0281f06444885c12720ce5716e68d876866e42a7.